CanvasColorSpaceProposal.md

```diff
@@ -63,56 +104,72 @@ partial interface CanvasRenderingContext2D {
 
 Example:
 <pre>
-canvas.getContext('2d', { colorSpace: "linear-srgb", pixelFormat: "float16"});
+canvas.getContext('2d', { colorSpace: "rec2020",
+                          colorEncoding: "float16",
+                          colorLinearity: "radiance"} );
 </pre>
 
-#### Common properties of all color spaces
+#### The colorSpace canvas creation parameter
+
+* Color spaces match their respective counterparts as defined in the [CSS colorspaces](https://www.w3.org/TR/css-color-4/#predefined).
 * All input colors (e.g, fillStyle or strokeStyle, and gradient stops) follow the same interpretation as CSS color literals, regardless of canvas color space.
-* Images with no color profile, when drawn to the canvas, are assumed to already be in sRGB color space.
-* Unless otherwise explicitly specified by the user, toDataURL/toBlob will produce resources in sRGB color space. If the encoding format supports colorspace tagging or embedded color profiles, the resource will be tagged as being in sRGB color space.
-* Values written to WebGL drawing buffers (e.g, values written to gl_FragColor or the clear color) are defined to be in the output canvas color space.
+* Values written to WebGL backbuffers (e.g, values written to gl_FragColor or the clear color) are in the canvas's color space and linearity.
```

Comment:
Does this mean you are planning to patch people's shaders to do color transforms - that is one set of primaries to another? The hardware only helps with the "linearity".